scripts/typeahead.js

```diff
@@ -226,6 +227,7 @@ const searchOptions = [
     display: 'Pokérus',
     type: 'Pokérus',
     page: '',
+    redirects: ['EVs', 'Effort Values'],
```

Comment:
How far do we want to go with the redirecting/helping people to get to the right page? Should this also have `'Contagious', 'Infected', 'Resistant'`?

Reply:
Sure, doesn't hurt. Added